definitions/npm/react-apollo_v2.x.x/flow_v0.58.x-/react-apollo_v2.x.x.js

```diff
@@ -294,7 +294,7 @@ declare module 'react-apollo' {
     update?: (cache: DataProxy, mutationResult: FetchResult) => any,
     ignoreResults?: boolean,
     optimisticResponse?: Object,
-    refetchQueries?: (mutationResult: FetchResult) => Array<{query: DocumentNode, variables: TVariables}>,
+    refetchQueries?: (mutationResult: FetchResult) => Array<{query: DocumentNode, variables: {[string]: any}}>,
```

Comment:
It looks like this will also just take an array of strings or an array of `{query: DocumentNode, variables: {[string]: any}}`, rather than just the function (https://github.com/apollographql/react-apollo/blob/master/src/Mutation.tsx#L59).